addon/components/bootstrap-debounce-input.js

```diff
@@ -2,36 +2,34 @@ import Ember from 'ember';
 import layout from '../templates/components/bootstrap-debounce-input';
 import InputableMixin from '../mixins/components/inputable';
 import computedActionKey from '../utils/computed-action-key';
+import { task, timeout } from 'ember-concurrency';
 
 export default Ember.Component.extend(InputableMixin, {
   tagName: '',
   classNames: '',
   layout: layout,
 
-  value: "",
-  valueText: "",
-  debounce: 1000,
+  /* Public Attrs */
+  value: null,
+  debounce: 800,
```

Comment:
This changes the default and should probably update the README or remove it from the README.